apps/mobile/ios/Podfile

```
@@ -17,6 +17,131 @@ install! 'cocoapods',
+    system("cd '#{go_tmp_dir}' && tar -xzf go.tar.gz")
+    
+    # Set Go environment variables for this session
+    ENV['GOROOT'] = go_install_dir
+    ENV['PATH'] = "#{go_install_dir}/bin:#{ENV['PATH']}"
+    
+    puts "Go installed successfully to #{go_install_dir}"
+    puts "Note: You may need to add #{go_install_dir}/bin to your PATH permanently"
+  else
+    puts "Go is already installed: #{go_version}"
+  end
+end
```

RuboCop (1.80.2)

[convention] Assignment Branch Condition size for `installGo` is too high. [<10, 22, 9> 25.79/23]

(Metrics/AbcSize)